CODEOWNERS

```diff
@@ -5,4 +5,4 @@
 
 # Default owner
 * @pq-code-package/pqcp-hackers
```

Comment:
github complains:
```
Unknown owner on line 7: make sure the team @pq-code-package/pqcp-hackers exists, is publicly visible, and has write access to the repository
* @pq-code-package/pqcp-hackers
```

Reply:
I fixed this in https://github.com/pq-code-package/mlkem-c-aarch64/pull/41